Remove deprecated components and props for V3 (#995)
sumup-oss/circuit-ui: .changeset/gentle-glasses-pretend.md

---
'@sumup/design-tokens': major
---

> Removed the `drawer` and `select` zIndex values from the design tokens. You can use `input` instead of `select`.

Reviewer: ```suggestion
Removed the `drawer` and `select` z-index values from the design tokens. Use `input` instead of `select`.
```
